src/Request/AbstractRequest.php

```diff
@@ -77,6 +98,11 @@ public function getParameters()
             $parameters['NUMAPPEL'] = $this->getCallNumber();
         }
 
+        // Direct Plus requests special case.
+        if (null !== $this->getSubscriberRef()) {
+            $parameters['REFABONNE'] = $this->getSubscriberRef();
```

Comment:
This not planned at the moment. Personally, I don't really see the interested of that because it's just an array for the request.

Feel free to open an issue/PR to discuss about that. :+1: 

If we continue the discussion on this PR, we will lose it because unrelated.